crates/provider-keyvalue-nats/src/lib.rs

```diff
@@ -70,14 +70,12 @@ impl KvNatsProvider {
             .await
             .context("failed to run provider")?;
         let connection = get_connection();
-        serve_provider_exports(
-            &connection.get_wrpc_client(connection.provider_key()),
-            provider,
-            shutdown,
-            bindings::serve,
-        )
-        .await
-        .context("failed to serve provider exports")
+        let wrpc = connection
+            .get_wrpc_client(connection.provider_key())
+            .await?;
```

Comment:
Will be good to note in the next provider-sdk release that this is a breaking change + all you need to do is `.await?` the creation of the client

Reply:
Yes, is there a changelog I should update?

Reply:
We autogenerate changelogs so I suppose not, We'll just have to remember ™️